### PR TITLE
Updated the postman collection

### PR DESCRIPTION
**Pull request by yashodgayashan**
**Added functions in the postman collection**
* postApplication
* deleteApplicationById
* updateApplicationById

### CODE REVIEW

File: core/src/env_platform/resources/environment_platform.postman_collection.json [hunk 1]
```diff
@@ -35,7 +35,62 @@
...
+					"path": [
+						"applications",
+						"tcf-20200513"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "putApplicationById",
```
Comment (deshankoswatte) [inline core/src/env_platform/resources/environment_platform.postman_collection.json:63]:
```suggestion
			"name": "updateApplicationById",
```

File: core/src/env_platform/resources/environment_platform.postman_collection.json [hunk 2]
```diff
@@ -35,7 +35,62 @@
...
+						"tcf-20200513"
+					]
+				}
+			},
+			"response": []
 		}
 	],
 	"protocolProfileBehavior": {}
-}
+}
```
Comment (deshankoswatte) [inline core/src/env_platform/resources/environment_platform.postman_collection.json:96]: Add new line at end of file if there is no problem.
```suggestion
}

```

File: core/src/env_platform/resources/environment_platform.postman_collection.json [hunk 3]
```diff
@@ -1,41 +1,73 @@
...
-	],
-	"protocolProfileBehavior": {}
+  "info": {
+    "_postman_id": "1645c04d-e462-4ea9-b5cf-77c92c6c6546",
+    "name": "environment_platform",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+  },
+  "item": [
+    {
+      "name": "postApplication",
```
Comment (deshankoswatte) [inline core/src/env_platform/resources/environment_platform.postman_collection.json:9]:
```suggestion
      "name": "saveApplication",
```

**Comment by yashodgayashan** [comment]
@sanjulamadurapperuma any concerns

**Pull request merged**